node/coinstacks/bnbsmartchain/api/src/app.ts

```diff
@@ -0,0 +1,83 @@
+import express, { json, urlencoded } from 'express'
+import cors from 'cors'
+import { join } from 'path'
+import { Server } from 'ws'
+import swaggerUi from 'swagger-ui-express'
+import {
+  evm,
+  middleware,
+  ConnectionHandler,
+  Registry,
+  BlockHandler,
+  TransactionHandler,
+} from '@shapeshiftoss/common-api'
+import { Tx as BlockbookTx, WebsocketClient, getAddresses, NewBlock } from '@shapeshiftoss/blockbook'
+import { Logger } from '@shapeshiftoss/logger'
+import { service } from './controller'
+import { RegisterRoutes } from './routes'
+
+const PORT = process.env.PORT ?? 3000
+const INDEXER_WS_URL = process.env.INDEXER_WS_URL
+
+if (!INDEXER_WS_URL) throw new Error('INDEXER_WS_URL env var not set')
+
+export const logger = new Logger({
+  namespace: ['unchained', 'coinstacks', 'bnbsmartchain', 'api'],
+  level: process.env.LOG_LEVEL,
+})
+
+const app = express()
+
+app.use(json())
+app.use(urlencoded({ extended: true }))
+app.use(cors())
+
+app.get('/health', async (_, res) =>
+  res.json({ status: 'up', asset: 'bnbsmartchain', connections: wsServer.clients.size })
+)
+
+const options: swaggerUi.SwaggerUiOptions = {
+  customCss: '.swagger-ui .topbar { display: none }',
+  customSiteTitle: 'ShapeShift BNB Smart Chain API Docs',
```

Comment:
I would probably keep the official `Binance Smart Chain` terminology at least here since this is user-facing docs stuff

Reply:
https://www.binance.com/en/blog/ecosystem/introducing-bnb-chain-the-evolution-of-binance-smart-chain-421499824684903436